config/initializers/time_formats.rb

```diff
@@ -6,3 +6,5 @@
 Time::DATE_FORMATS[:weekday_day_month_year_at_time] = "%A %e %B %Y at %l:%M%P"
 # For example: 2 December 2019
 Time::DATE_FORMATS[:day_month_year] = "%d %B %Y"
+# For example: 2019-12-02
+Time::DATE_FORMATS[:reports_year_month_day] = "%Y-%m-%d"
```

Comment:
I'd just suggest a slight tweak on the name. I don't think we need to refer to where this is currently used. Plus it means it's inconsistent with the other formats. So I'm suggesting just stick with a name that describes the format itself.

```ruby
Time::DATE_FORMATS[:year_month_day] = "%Y-%m-%d"
# or
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```

Reply:
```suggestion
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```

Reply:
```suggestion
Time::DATE_FORMATS[:year_month_day_hyphens] = "%Y-%m-%d"
```